pkg/descriptions.json

```diff
@@ -1997,18 +1997,11 @@
     "Name": "System.Text.Json",
     "Description": "Provides a fast, non-cached, forward-only way to read UTF-8 encoded JavaScript Object Notation (JSON) text.",
```

Comment:
We should update the description too to make it broader, rather than specific to the reader.

Reply:
Do you have a suggestion? I'm not very good at writing descriptions 😄 Perhaps something like:
```
Provides types that help read/write/serialize/deserialize UTF-8 encoded JavaScript Object Notation (JSON) text.
```

Reply:
I took a look at some of the XML namespaces and their description.
> "Provides classes for serializing objects to the Extensible Markup Language (XML) and deserializing XML data to objects."

https://github.com/dotnet/corefx/blob/c53300f0696c4817bc06a8ef8711a0bb46f362ec/pkg/descriptions.json#L2199

> "Provides an older in-memory Extensible Markup Language (XML) programming interface that enables you to modify XML documents. Developers should prefer the classes in the System.Xml.XDocument package."

> "Provides classes for serializing objects to the Extensible Markup Language (XML) and deserializing XML data to objects."

**Maybe something like this (if it is not too verbose):**
Provides types to serialize objects to JavaScript Object Notation (JSON) text and deserialize JSON data to objects. Also provides types to read and write JSON encoded as UTF-8, and to create an in-memory document object model (DOM) that is read-only, to get random access of JSON elements within a structured view of the JSON.